aws_database/db_service/models.py

```diff
@@ -0,0 +1,136 @@
+from sqlalchemy import Column, ForeignKey, Integer, String, Boolean, Numeric, DateTime, Text, Date, JSON
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.dialects.postgresql import DOUBLE_PRECISION, TIMESTAMP, JSONB
+
+Base = declarative_base()
+
+class AssessmentResult(Base):
+    __tablename__ = 'assessmentResult'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    assessment_id = Column(Integer, ForeignKey('assessment.id'), nullable=False)
+    score = Column(Numeric)
+    flag = Column(Boolean)
+    note = Column(Text)
+    vref = Column(Text, ForeignKey('verseReference.fullverseid'))
+    source = Column(Text)
+    target = Column(JSONB)
+    hide = Column(Boolean)
+    book = Column(Text)
+    chapter = Column(Integer)
+    verse = Column(Integer)
+
+    assessment = relationship("Assessment", back_populates="assessment_results")
+
+class Assessment(Base):
+    __tablename__ = 'assessment'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    revision = Column(Integer, ForeignKey('assessment.revision'))
+    reference = Column(Integer, ForeignKey('assessment.reference'))
+    type = Column(Text)
+    status = Column(Text)
+    requested_time = Column(TIMESTAMP)
+    start_time = Column(TIMESTAMP(timezone=True))
+    end_time = Column(TIMESTAMP(timezone=True))
+
+    assessment_results = relationship("AssessmentResult", back_populates="assessment")
+
+class BibleRevision(Base):
+    __tablename__ = 'bibleRevision'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    date = Column(Date)
+    bibleversion_id = Column(Integer, ForeignKey('bibleVersion.id'), nullable=False)
+    published = Column(Boolean)
+    name = Column(Text)
+    backtranslation_id = Column(Integer, ForeignKey('bibleVersion.id'))
+    machinetranslation = Column(Boolean)
+
+class BibleVersion(Base):
+    __tablename__ = 'bibleVersion'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(Text)
+    isolanguage_id = Column(Text, ForeignKey('isoLanguage.iso639'))
+    isoscript_id = Column(Text, ForeignKey('isoScript.iso15924'))
+    abbreviation = Column(Text)
+    rights = Column(Text)
+    forwardtranslation = Column(Integer)
```

Comment:
@woodwardmw We aren't using `forwardtranslation `and `backtranslation` here for `BibleVersion`, right?